contracts/svm/package.json

```diff
@@ -7,13 +7,14 @@
     "@coral-xyz/anchor": "^0.30.1"
   },
   "devDependencies": {
-    "chai": "^4.3.4",
-    "mocha": "^9.0.3",
-    "ts-mocha": "^10.0.0",
+    "@kamino-finance/limo-sdk": "^0.2.3",
```

Comment:
is it compatible with our anchor version. Not sure if it matters btw :D

Reply:
I don't think it does